packages/ilspy.vm/ilspy.vm.nuspec

```
@@ -2,14 +2,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ilspy.vm</id>
-    <version>9.1.0</version>
+    <version>9.1.0.20250505</version>
     <authors>SharpDevelop Team</authors>
-    <description>The open-source .NET assembly browser and decompiler.</description>
+    <description>ILSpy is a open-source .NET assembly browser and decompiler.</description>
```

Do not include license information in the description.
```suggestion
    <description>ILSpy is a .NET assembly browser and decompiler.</description>
```